app/src/test/java/net/squanchy/service/firestore/FirestoreEventMapperTest.kt

```diff
@@ -0,0 +1,136 @@
+package net.squanchy.service.firestore
+
+import net.squanchy.eventdetails.domain.view.ExperienceLevel
+import net.squanchy.schedule.domain.view.Event
+import net.squanchy.support.lang.Checksum
+import org.joda.time.DateTimeZone
+import org.joda.time.LocalDateTime
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.MockitoAnnotations
+import java.util.Date
+
+private const val FAKE_STRING = "eventId"
+private const val FAKE_TYPE = "social"
+private const val FAKE_EXPERIENCE_LEVEL = "intermediate"
+private const val CHECKSUM_RESULT = 1000L
+
+class FirestoreEventMapperTest {
+
+    @Mock
+    lateinit var checksum: Checksum
+
+    lateinit var timeZone: DateTimeZone
+
+    @Before
+    fun before() {
+        MockitoAnnotations.initMocks(this)
```

Comment:
We should use the `@Rule` instead! See https://static.javadoc.io/org.mockito/mockito-core/2.13.0/org/mockito/junit/MockitoRule.html (or other tests in the project)